### DISCUSSION TITLE
打包部署后点击左侧菜单请求文件js时，request headers 没有携带referer

### DISCUSSION THREAD

**Discussion by kk305cs** [vuepress/core#1225] [Q&A]
由于项目服务有防盗链配置，打包部署后，点击左侧菜单时，network 请求文件 js 时，request headers 没有携带 referer，点击左侧菜单会打不开对应链接，并且 console 内报错 403 阻止访问该 js 文件
![image](https://user-images.githubusercontent.com/26682400/210937027-b0307edf-95ee-433b-b60d-8b3bb8c69ba3.png)

**Comment by kk305cs** [comment]
官网有一个链接打开请求的js 也没有携带referer
![image](https://user-images.githubusercontent.com/26682400/210944322-8754d5ac-83f8-46ce-88ad-80cacbb103ed.png)

**Reply by meteorlxy** [reply]
I can see the referer in my browser. Could you reproduce it now?
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/18205362/223986645-ebe9a998-3366-4952-8be0-439ce7be4e83.png">

**Comment by Mister-Hope** [comment]
CC @meteorlxy

**Reply by kk305cs** [reply]
已找到问题所在，如果两个以上的 md 文件引入同一张图片就会出现此问题，需保证图片的唯一性，我处理的办法是将图片复制后重命名，即可，若大家遇到此问题可参考此方法解决。
